core/src/main/java/org/apache/carbondata/core/datamap/DataMapFilter.java

```diff
@@ -72,6 +85,17 @@ public Expression getNewCopyOfExpression() {
     }
   }
 
+  public DataMapFilter(SegmentProperties properties, CarbonTable table, Expression expression) {
```

Comment:
Move this constructor above getNewCopyOfExpression method

Reply:
done